### DISCUSSION TITLE
Development roadmap

### DISCUSSION THREAD

**Discussion by VeroFess** [VeroFess/PalWorld-Server-Unoffical-Api#3] [General]
Currently, I hope to prioritize addressing the issue of cheats. In other words, I will first provide the following events:

- Player login and logout events,
- Allow interception and modification of most RPCs (as a very primitive temporary solution),
- Allow plugins to audit and intercept when the server triggers build or destruction events.

**Comment by Minidoracat** [comment]
希望可以取得登入當前的 IP 或 steam ID
不過外掛能夠偽裝 steam ID，偽裝玩家存檔，所以能取得 IP 是最好
這樣至少能夠做一個白名單系統
